### PR TITLE
Added large seashell

### PR DESCRIPTION
**Pull request by EPuncker**
Script by @zxmatzx on otland, I've just changed a few things to make it "compliant" to others  TFS scripts

### CODE REVIEW

**Comment by TheSumm** [comment]
We should probably decide on a way to define constants for storage keys that are used within the core scripts before adding more scripts with arbitrary storage keys.

File: data/actions/scripts/other/largeseashell.lua [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+			msg = "You found a beautiful pearl."
+		else
+			msg = "Nothing is inside."
+		end
+		player:say(msg, TALKTYPE_MONSTER_SAY, false, player, item:getPosition())
+		item:transform(7553)
+		player:setStorageValue(43000, os.time() + 72000)
+		item:getPosition():sendMagicEffect(CONST_ME_BUBBLES)
+		return true
+	else
```
Comment (TheSumm) [inline data/actions/scripts/other/largeseashell.lua]: `else` is not needed since you are returning in the if-block.

File: data/items/items.xml [hunk 2]
```diff
@@ -13166,6 +13166,8 @@
 	</item>
 	<item id="7553" article="a" name="large seashell">
 		<attribute key="description" value="It is empty." />
+		<attribute key="decayTo" value="7552"/>
+		<attribute key="duration" value="900"/>
```
Comment (TheSumm) [inline data/items/items.xml]: AFAIK it is only 5 minutes, but I am not exactly sure.

File: data/actions/scripts/other/largeseashell.lua [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+		player:say("You have already opened a shell today.", TALKTYPE_MONSTER_SAY, false, player, item:getPosition())
+		return false
```
Comment (TheSumm) [inline data/actions/scripts/other/largeseashell.lua]: Should be `return true`

File: data/actions/scripts/other/largeseashell.lua (hunk 1 above)
Comment (EPuncker) [reply data/actions/scripts/other/largeseashell.lua]: 2019-12-31 23_00_07-Window
this then?

File: data/global.lua [hunk 4]
```diff
@@ -2,6 +2,7 @@ math.randomseed(os.time())
 dofile('data/lib/lib.lua')
 
 STORAGEVALUE_PROMOTION = 30018
+DELAY_LARGE_SEASHELL = 30019
```
Comment (DSpeichert) [inline data/global.lua]: The constant name says nothing about the type, i.e. it should be named `STORAGEVALUE_DELAY_LARGE_SEASHELL`

**Pull request merged**